en/docs/get-started/tutorials/secure-an-api-with-role-based-authorization.md

```diff
@@ -0,0 +1,231 @@
+     Similarly, add `get_user`, `create_user`, and `delete_user` as permission values.
+4. Select the permission values as follows for each of the resources:
+
+    ![Permission values](../../assets/img/tutorials/role-based-auth/permission-values.png){.cInlineImage-full}
+
+5. Click **Save and Deploy**. 
+Now you are ready to promote the API to production.
+6. In the left navigation menu, click **Deploy**.
+7. Go to the **Development** card and click **Promote**.  The **Configure & Deploy** pane opens with `Use Development Endpoint Configuration` selected by default.
+8. Click **Next** to promote the API to production. Now you can publish the API.
+  
+    !!!note
+
+            To verify whether the permission(scope) values have been applied to the resources, click **Settings**, then click the **Resources** tab, expand a resource, and check the attached permission value.
+
+            ![Resource permission](../../assets/img/tutorials/role-based-auth/resource-permission.png){.cInlineImage-full}
+
+9. In the **Manage** view, click **Lifecycle**.
+10. In the **Lifecycle Management** pane, click **Publish**.
+    A message appears where you can specify whether you want to publish a connector for this REST API proxy. Creating a connector for this REST API proxy makes it available in the Marketplace. 
+11. Click **No, Thanks**. This changes the status of the API to `Published`.
+
+Now the REST API proxy is ready to be consumed. An application developer can discover the API, subscribe to it and then invoke it.
+
+### Step 3: Subscribe to the Published API
+
+In this step, you take the role of an application developer who consumes the published REST API.
+To consume the REST API, you must create an application, generate keys, and subscribe to the API.
+
+Follow these steps:
+
+1. Go to the [API Developer Portal](https://devportal.choreo.dev/) and click **Applications** on the top menu.
+2. Click **Create**.
+3. Enter `User Management App` as the **Application Name**, select **10PerMin** as the **Per Token Quota**, and then click **Create**. This creates the application and takes you to the application overview page.
+4. On the left navigation menu, click **Production Keys**.
+5. Click to expand **Advanced Configurations** and follow these steps to generate credentials for the application:
+    1. Select **Code** as the grant type.
+    2. Enter the hosted URL of the application as the **Callback URL**.
+    3. Click **Generate Credentials**.
+6. To Subscribe to the API follow these steps:
+    1. On the left navigation menu, click **Subscriptions**.
+    2. In the Subscription Management pane, click Add APIs.
+    3. In the **Add APIs** list, go to the API you created, and click **Add**.
+
+### Step 4: Define roles and assign them to groups
+
+Follow these steps:
+
+1. Click **Manage Permission** on the top right. This takes you to the **Roles** tab in Asgardeo, where you can add applicable roles.
```

Comment:
It's in the Developer Portal itself. Did not specifically mention it here because the user is already in the Developer Portal at this point. Mentioned it here too to avoid any confusion.